### DISCUSSION TITLE
getting layerrule to use my custom animation

### DISCUSSION THREAD

**Discussion by bohanubis** [hyprwm/Hyprland#5173] [Q&A] [answered]
this should probably work but I'm not sure thats why I am here after all
```
bezier = rofi_curve,0.34, -0.09, 0, 0.96
animation = rofi_anim,1,1,rofi_curve,popin
layerrule = animation rofi_anim,rofi
```
this works though
```
layerrule = animation popin,rofi
```
so yeah is this expected ??

also the error message is no such animation so the problem is in this line
```
animation = rofi_anim,1,1,rofi_curve,popin
```

**Comment by vaxerski** [comment]
that's not how the thing works, read the wiki please. https://wiki.hyprland.org/Configuring/Animations/ and https://wiki.hyprland.org/Configuring/Window-Rules/#layer-rules

**Comment by bohanubis** [comment, marked as answer]
https://github.com/hyprwm/Hyprland/discussions/5106
I found that basically what I want so yeah I guess yeah sure sorry vaxry for annoyance
